src/screens/Booking/Booking.tsx

```diff
@@ -0,0 +1,53 @@
+// @ts-nocheck
+
+import React from 'react';
+import {View, ScrollView} from 'react-native';
+import {RouteProp, useRoute} from '@react-navigation/native';
+import {FlightData} from '@components';
+import Baggage from './components/Baggage/Baggage';
+import Form from './components/PassengerForm/Form';
+import styles from './Booking.styles';
+import {RootStackParamsList} from '@navigation/types';
+
+const Booking = () => {
+  const route = useRoute<RouteProp<RootStackParamsList, 'BookingScreen'>>();
```

Comment:
Route is available on component's(Booking) props. There shouldn't be any need for `useRoute`